pkg/controller/namespace/namespace_controller.go

```diff
@@ -146,7 +146,6 @@ func (r *ReconcileNamespace) Reconcile(request reconcile.Request) (reconcile.Res
 		// check for errors, but ignore when rolebinding already exists
 		if err != nil && !errors.IsAlreadyExists(err) {
 			reqLogger.Info("Error creating rolebinding", "RoleBinding", rb.Name, "Error", err)
-			return reconcile.Result{}, err
```

Comment:
This isn't right. Think like.. api server timeout. It should be able to requeue and start the process over again.

/hold

Reply:
Is there an easy way to requeue without altering the object? This controller watches namespaces, so we don't want to update them when we create the role bindings.

Reply:
Because you're returning a non-nil err here, the controller should automatically requeue. Is it not?

Reply:
Hmm I didn't know that.

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/524b614e5a34c80432744688e1a90a0f3b155780/pkg/internal/controller/controller.go#L216-L221

Reply:
:+1: